sigopt/xgboost/constants.py

```diff
@@ -31,3 +32,37 @@
   **dict(zip(CLASSIFICATION_METRIC_CHOICES, ['maximize']*len(CLASSIFICATION_METRIC_CHOICES))),
   **dict(zip(REGRESSION_METRIC_CHOICES, ['minimize']*len(REGRESSION_METRIC_CHOICES))),
 }
+
+# Note: only the XGB general params. Omitted monotone_constraints and interaction_constraints b/c more complex.
+# Also omitting refresh_leaf b/c it's a boolean value
+PARAMETER_INFORMATION = {
+
+  # Numerical Values
+  'alpha':              {'type': 'double', 'limits': '[0, Inf]'},
+  'colsample_bylevel':  {'type': 'double', 'limits': '(0, 1]'},
+  'colsample_bynode':   {'type': 'double', 'limits': '(0, 1]'},
+  'colsample_bytree':   {'type': 'double', 'limits': '(0, 1]'},
+  'eta':                {'type': 'double', 'limits': '(0, 1]'},
```

Comment:
I noticed that xgboost silently allows eta > 1. Not sure if they cap it internally or something.

Reply:
Fair. In our meta learning I think we actually let eta < 10 so yea doable. I'll set the upper bound to be inf.